src/main/webapp/index.html

```diff
@@ -10,7 +10,8 @@
 <div class="container">
   <h1 class="display-2 mb-5 mt-5 text-center">The Open Source Community is for <span class="emphasis">Everyone</span></h1>
   <!-- TODO: Change button text/link depending on user authorization. -->
-  <div class="d-flex justify-content-center"><div class="btn btn-emphasis mt-5 mb-5 text-center btn-lg btn-block btn-main">Create a Profile!</div></div>
+  <!-- <div class="btn btn-emphasis mt-5 mb-5 text-center btn-lg btn-block btn-main">Create a Profile!</div> -->
+  <div id="button-container" class="d-flex justify-content-center"></div>
```

Comment:
nit: It might be worth making this id a bit more specific (keep in mind you would need to change it in the JS below too)
```suggestion
  <div id="auth-button-container" class="d-flex justify-content-center"></div>
```